File: quiz/src/main/java/com/example/quiz/service/question/QuestionService.java
package com.example.quiz.service.question;

import com.example.quiz.base.baseInterface.BaseService;
import com.example.quiz.model.entity.question.Question;
import com.example.quiz.model.entity.question.QuestionRequest;
import com.example.quiz.model.entity.question.QuestionResponse;
import com.example.quiz.model.entity.question.QuestionView;

public interface QuestionService extends BaseService<Question, Long, QuestionRequest, QuestionResponse, QuestionView> {
}

File: quiz-fe/src/share/services/question/question.service.ts
import { ENV } from "@/share/utils/env";
import { BaseService } from "../BaseService";
import { Question, QuestionRequest, QuestionResponse, QuestionView } from "./models";

export class QuestionService extends BaseService<
Question,
QuestionRequest,
QuestionResponse,
QuestionView
> {
     constructor() {
          super(ENV.API_URL, 'questions');
     }
}

export const questionService = new QuestionService();